add-on/src/popup/browser-action/gateway-status.js

```diff
@@ -21,10 +21,9 @@ export default function gatewayStatus ({
   gatewayAddress,
   gatewayVersion,
   ipfsApiUrl,
-  ipfsNodeType,
   swarmPeers
 }) {
-  const api = ipfsApiUrl && ipfsNodeType === 'embedded' ? 'js-ipfs' : ipfsApiUrl
+  const api = ipfsApiUrl
```

Comment:
Below, we use this value in `panel_statusApiAddress`. Is that read from anything?